spec/models/cart_item_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe CartItem, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Empty test file?

Reply:
Auto generated, will have a look.

Reply:
Yes, Rails generally generates the spec files for us so we don't forget our unit tests.